MIDIchlorians/MIDIchlorians/AppDelegate.swift

```diff
@@ -148,10 +149,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         appSamples.forEach { sample in
             // if saving fails, what are we gonna do?
```

Comment:
Should probably remove this comment if we are not doing anything to handle failure :)